<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Title</class>
 <widget class="QMainWindow" name="Title">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1146</width>
    <height>845</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="ProgramTitle">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>60</y>
      <width>601</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>QRCODE MANAGEMENT SYSTEM</string>
    </property>
   </widget>
   <widget class="QLabel" name="RegPepsi">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>260</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>PEPSI:</string>
    </property>
   </widget>
   <widget class="QLabel" name="RegValuePepsi">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>270</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="RegValueFanta">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>330</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="RegFanta">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>320</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>FANTA:</string>
    </property>
   </widget>
   <widget class="QLabel" name="RegValueCoca">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>210</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="RegCoca">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>200</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>COCA:</string>
    </property>
   </widget>
   <widget class="QLabel" name="RegValueSum">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>380</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="RegCoca_2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>370</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>SUM:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="StartButton">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>480</y>
      <width>261</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="text">
     <string>START</string>
    </property>
   </widget>
   <widget class="QPushButton" name="StopButton">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>620</y>
      <width>261</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>26</pointsize>
     </font>
    </property>
    <property name="text">
     <string>STOP</string>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="CameraDisplay">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>140</y>
      <width>691</width>
      <height>471</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Camera Display Here:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>670</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Camera Switch: </string>
    </property>
   </widget>
   <widget class="QPushButton" name="OnCamButton">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>710</y>
      <width>171</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>ON</string>
    </property>
   </widget>
   <widget class="QPushButton" name="OffCamButton">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>710</y>
      <width>171</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>OFF</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>600</y>
      <width>201</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Detected Data:</string>
    </property>
   </widget>
   <widget class="QLabel" name="DetectedData">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>620</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Product Name</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1146</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
